mybooks/oauth_views.py

```
@@ -99,7 +96,7 @@ def apps(request):
 
     # Pre-fill registration data for new application
     registration_data = {
-        "client_name": f"My Books Agent",
+        "client_name": f"My Books Client",
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)